README.md

```diff
@@ -34,7 +35,7 @@ We forked the open source project [django-job-portal](https://github.com/manjuru
 
 **Anem per feina** is a collaborative effort where everybody is more than welcome to contribute, no experience is required!
 
-Have a look at [CONTRIBUTING.md](CONTRIBUTING.md) file that describes the process to submit a contribution, and come say hi 👋👋 to [GeeksCat slack](https://geekscat.slack.com) where most contributors hang out.
+Have a look at [CONTRIBUTING.md](CONTRIBUTING.md) file that describes the process to submit a contribution, support translations, and come say hi 👋👋 to [GeeksCat slack](https://geekscat.slack.com) where most contributors hang out.
```

Comment:
I think this change is not required. Supporting translations itself is a contribution.

If you want to specify that in order to support the translations you should read the contribution guide first, I think you should probably add a new sentence for it. As it is written right now it looks a bit weird.

Reply:
Thanks @ytturi I'll, fixed :)